### ISSUE TITLE
Adding Authentication Token for api requests

### ISSUE THREAD

**Issue by annapurna** [graphiti-api/vandal_ui#10]
How to pass an authentication token to api requests here?

**Comment by yjukaku** [comment]
I'm also interested in knowing about this.

**Comment by richmolj** [comment]
I use [request.ly](https://www.requestly.in/) which has the added benefit of being able to hit the endpoints directly with your browser

**Comment by XHPJ** [comment]
Hi @richmolj I would like to implement a token support for vandal but right now I am a bit clueless on how to do this. I saw the issue #4 and there it was suggested to start in the query.ts file https://github.com/graphiti-api/vandal/blob/b3fa13366316e3c63f6e9b466d576d600fab2744/src/query.ts#L110. Would this be all on the vandal side?

**Comment by richmolj** [comment]
Yep you got it. That's where we query the server. So we need some UI to accept the header, and then alter `query.ts` to send that header when making API requests
